etc/calplus/calplus.conf.sample

```diff
@@ -0,0 +1,210 @@
+[DEFAULT]
```

Comment:
Actually, I think we should have it on our project.

For instance:
https://github.com/openstack/glance/tree/master/etc